### ISSUE TITLE
Trakt sync doesnt work

### ISSUE THREAD

**Issue by weeebdev** [Stremio/stremio-core#394]
Trakt sync doesnt work on my android tv.

**Comment by elpiel** [comment]
Thanks for reporting this!
Are you referring to a feature in the android TV app itself, because this is not present (yet) in the app itself.

Are you using an addon for Trakt sync (by dexter) by any chance?

**Comment by weeebdev** [comment]
Yep. As far as I know, ATV doesn't have trakt feature (at least I didn't find it). I authorized via Mobile app (on Android). Also, it scrobbles the movie only after the credits. Can we change it, so that the movie will be scrobbled at, let's say, 80% of the video??
